xla_native_functions.yaml

```diff
@@ -107,6 +107,8 @@ supported:
   - _adaptive_avg_pool3d_backward
   - _amp_foreach_non_finite_check_and_unscale_
   - _amp_update_scale_
+  - copy
```

Comment:
nit: I believe these two should be already there after the rebase?

Reply:
Let me remove.